examples/src/writeread.c

```diff
@@ -317,34 +341,50 @@ int main(int argc, char* argv[])
 
     write_bw = bandwidth_mib(rank_bytes, time_wr.elapsed_sec);
     aggr_write_bw = test_reduce_double_sum(cfg, write_bw);
-    max_write_time = test_reduce_double_max(cfg, time_wr.elapsed_sec);
+    max_write_time = test_reduce_double_max(cfg, time_wr.elapsed_sec_all);
 
     min_sync_time = test_reduce_double_min(cfg, time_sync.elapsed_sec);
     max_sync_time = test_reduce_double_max(cfg, time_sync.elapsed_sec);
 
     read_bw = bandwidth_mib(rank_bytes, time_rd.elapsed_sec);
     aggr_read_bw = test_reduce_double_sum(cfg, read_bw);
-    max_read_time = test_reduce_double_max(cfg, time_rd.elapsed_sec);
+    max_read_time = test_reduce_double_max(cfg, time_rd.elapsed_sec_all);
 
     if (test_config.rank == 0) {
         size_t total_bytes = rank_bytes * test_config.n_ranks;
         double eff_write_bw, eff_read_bw;
         eff_write_bw = bandwidth_mib(total_bytes, max_write_time);
         eff_read_bw = bandwidth_mib(total_bytes, max_read_time);
 
-        printf("File create time is %.3lf s\n",
+        double write_sync_bw = bandwidth_mib(total_bytes,
```

Comment:
@adammoody should this use `max_write_time` and `max_sync_time` instead of the local timer values?

Reply:
Good question.  For sync time, the writeread test is now reporting:
```
Min Sync Time = allreduce(MIN, time_sync.elapsed_sec)
Max Sync Time = allreduce(MAX, time_sync.elapsed_sec)
Global Sync Time = time_sync.elapsed_sec_all value from rank 0
```
It's useful to keep all three of those, because the min/max shows the spread of local sync cost across processes, which can be significant, and the global cost shows the cost from before the first process starts until the last process ends.

I could update the max_sync_time to do an allreduce of the elapsed_sec_all value instead, and add new min_local_sync_time and max_local_sync_time variables to capture those.